scripts/build/check_init_priorities.py

```diff
@@ -30,7 +30,6 @@
 
 # This is needed to load edt.pickle files.
 sys.path.insert(0, os.path.join(os.path.dirname(__file__), "..", "dts", "python-devicetree", "src"))
-from devicetree import edtlib  # pylint: disable=unused-import
```

Comment:
The warning was disabled for pylint, maybe this is still needed? CC @fabiobaltieri